modules/core/src/v2/coins/trx.ts

```diff
@@ -202,11 +213,50 @@ export class Trx extends BaseCoin {
   signMessage(key: KeyPair, message: string | Buffer): Buffer {
     const toSign = this.toHexString(message);
 
-    let sig = tronWeb.Trx.signString(toSign, key.prv, true);
+    let sig = bitgoAccountLib.Trx.Utils.signString(toSign, key.prv, true);
 
     // remove the preceding 0x
     sig = sig.replace(/^0x/, '');
 
     return Buffer.from(sig, 'hex');
   }
+
+  /**
+   * Explain a Tron transaction from txHex
+   * @param params
+   * @param callback
+   */
+  explainTransaction(
+    params: ExplainTransactionOptions,
+    callback?: NodeCallback<TronTransactionExplanation>
+  ): Bluebird<TronTransactionExplanation> {
+    return co<TronTransactionExplanation>(function*() {
+      const txHex = params.txHex || (params.halfSigned && params.halfSigned.txHex);
+      if (!txHex || !params.fee) {
+        throw new Error('missing explain tx parameters');
+      }
+      const coinName = this.getChain();
+      const txBuilder = new TransactionBuilder({ coinName });
+      txBuilder.from(txHex, bitgoAccountLib.BaseCoin.TransactionType.Send);
```

Comment:
I think this will be deprecated as of Jorge's PR (transactionType will be removed)